dandiapi/api/tests/test_zarr_upload.py

```diff
@@ -152,7 +152,6 @@ def test_zarr_rest_upload_complete_missing_file(
 
     resp = authenticated_api_client.post(f'/api/zarr/{zarr_archive.zarr_id}/upload/complete/')
     assert resp.status_code == 400
-    assert resp.json() == [f'File {upload.path} does not exist.']
```

Comment:
Why is this removed?

Reply:
What was the reason?

Reply:
I believe it was because the changes in #1220 made it obsolete (@danlamanna can correct me if i'm wrong)